src/flatMap.ts

```diff
@@ -38,8 +38,16 @@ export function flatMap() {
   return purry(_flatMap, arguments, flatMap.lazy);
 }
 
+function _map<T, K>(array: Array<T>, mapFn: (value: T) => K): Array<K> {
+  const newArray = new Array<K>(array.length);
```

Comment:
I know unicorn eslint consider this a warning and suggest a different syntax. I don't know the reasoning behind this but perhaps it's worthwhile to conform to that:

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-new-array.md

Reply:
Due to the reasons explained on https://github.com/remeda/remeda/pull/286#discussion_r1140460891, I think it would be worth disabling that rule for this specific use case.

Reply:
I think the only real change is doing `Array.from<K>({ length: array.length })` instead of `new Array<K>(array.length)`. I don't think it's related to the comment you linked to